webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckNotificationService.scala

```diff
@@ -50,10 +50,10 @@ class HealthcheckNotificationService(
     v <- Ref.make[List[HealthcheckResult]](List.empty)
   } yield v).runNow
 
-  private[this] def reloadCache(ref: Ref[List[HealthcheckResult]]): UIO[Unit] = {
+  def reloadCache(ref: Ref[List[HealthcheckResult]] = healthcheckCache): UIO[Unit] = {
```

Comment:
why that change? 
Default parameters are a code smell, they tend to bring a lot of complicated problem on the long term. 
If you need a way to say "now, do init your internal things", it's better to have a dedicated `init` method. That way, it's also easier to understand you class state machine (entry points, transition, etc).